### PR TITLE
add make target to pull kabanero-operator:master

### PR DESCRIPTION
**Pull request by stevenschader**
add new make target to pull the operator from master/latest for automated testing

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -229,6 +229,11 @@ endif
 # Install Test
 int-test-install: creds build-image push-image int-install int-config
 
+int-master-install: creds  pull-master-image int-install int-config
+
+pull-master-image:
+	docker pull ${IMAGE}
```
[the remark this answers is not in the document]
Comment (stevenschader) [reply Makefile:235]: We want to pull the latest and greatest available from the master branch.  If that's 'latest' then great.  If so, then I can change the names to :

int-docker-image-install: creds  pull-master-image int-install int-config

pull-docker-image:
	docker pull ${IMAGE}

[the remark this answers is not in the document]
Comment (kaczyns) [reply Makefile]: I guess I should be more specific - latest is the latest build, not necessarily the latest code that is in master.  if someone builds a patch release on a previous level, that may also get pushed to `latest`.  I don't think `latest` is the tag that you want to use here.

**Comment by kaczyns** [comment]
@stevenschader please let me know if you plan to come back to this.... I don't think you want to use "latest" because it might not have the latest chronological code, it will just contain the latest thing that was built (which might be a fix pack on Kabanero 0.3.x).

**Comment by stevenschader** [comment]
I believe our effort has moved toward just using the install.sh, and not handcrafting up a build from docker images.

**Pull request closed without merging**